library/core/src/main/java/com/google/android/exoplayer2/mediacodec/MediaCodecUtil.java

```diff
@@ -318,7 +318,16 @@ private static boolean isCodecUsableDecoder(android.media.MediaCodecInfo info, S
     }
 
     // Work around https://github.com/google/ExoPlayer/issues/4519.
-    if ("OMX.SEC.mp3.dec".equals(name) && "SM-T530".equals(Util.MODEL)) {
+    if ("OMX.SEC.mp3.dec".equals(name) && ("GT-I9152".equals(Util.MODEL)
+        || "GT-P5220".equals(Util.MODEL)
+        || "SM-T231".equals(Util.MODEL)
+        || "SM-T530".equals(Util.MODEL))) {
+      return false;
+    }
+    if (("OMX.SEC.mp3.dec".equals(name) || "OMX.brcm.audio.mp3.decoder".equals(name))
+        && ("GT-I9515".equals(Util.MODEL)
```

Comment:
I think this one belongs in the block above. It's not a broadcom based device.

Reply:
Overlooked! Swapping GT-I9515 with GT-I9152